VEDA_OS01/models.py

```diff
@@ -233,20 +233,15 @@ class Course(models.Model):
     """
     Model for Course.
     """
-    course_name = models.CharField('Course Name', max_length=100)
+    course_name = models.CharField('Course Name', max_length=255)
 
     # TODO: Change Name (this is reversed)
-    course_hold = models.BooleanField('Course Active', default=False)
-    institution = models.CharField('Inst. Code', max_length=4)
-    edx_classid = models.CharField('Class ID', max_length=5)
+    course_hold = models.BooleanField('Course Active', default=True)
+    institution = models.CharField('Inst. Code', max_length=50)
+    edx_classid = models.CharField('Class ID', max_length=255)
```

Comment:
Since, we've visited these fields recently. I'd like to add help text or comment to tell what's the actual purpose/usage of these fields. It'll help others to understand it without any contributor.